docs/system-admin-guide/email/README.md

```diff
@@ -33,14 +33,15 @@ To adapt email notification settings, go to Email and choose *Email notification
 
 The frequency of sending e-mails per work package can be set in [this way](../system-settings/display-settings/#time-and-date-formatting,-aggregation-of-changes-in-activity).
 
-## Configure email header and email footer
+
+### Configure email header and email footer
 
 Configure your notification email header and footer which will be sent out for email notifications from the system.
 
 1. **Formulate header and/or footer** for the email notifications. These are used for all the email notifications from OpenProject (e.g. when creating a work package).
 2. **Choose a language** for which the email header and footer will apply.
 3. **Send a test email**.
-   Please note: This test email does *not* test the notifications for work package changes etc. Find out more on the FAQ page for emails.
+   Please note: This test email does *not* test the notifications for work package changes etc. Find out more in [this FAQ](../../../installation-and-operations/faq/#i-dont-receive-emails-test-email-works-fine-but-not-the-one-for-work-package-updates).
```

Comment:
The link to the FAQ currently does not work - but that's probably because it has not yet been committed.